### ISSUE TITLE
bundle.min.js:95 Uncaught TypeError: Cannot set property 'dir' of undefined 

### ISSUE THREAD

**Issue by Rohan-Doshi** [liferay/alloy-editor#765]
I am using hashHistory(react-router) in my app for routing. I have a tab where I need to use alloy-editor.
I referred [this](https://github.com/ipeychev/alloyeditor-react-component) repo to set it up. 
When I use it, I encountered some problems:

```
Uncaught TypeError: Cannot set property 'dir' of undefined
Uncaught TypeError: Cannot read property 'add' of undefined

Uncaught SyntaxError: Unexpected token <
index.html
```

Also in my network tab, it fetched the index.html file twice: as an script(js) and css.

I was able to narrow down the error to routing. When I changed from hashHistory to browserHistroy, it started to work fine(without any change related to alloy-editor part).
Is alloy-editor not compatible with hashHistory ?

**Comment by wincent** [comment]
Hi @Rohan-Doshi. Offhand I'm not aware of a reason why the routing mode could trigger an error like that. As this ticket is quite old now and we just released v2.0.0-beta.1 it is quite possible that the behavior you saw doesn't reproduce any more. I'm going to close this but please comment if you think it still applies and we can re-open it.

**Comment by alk-mandrianarijaona** [comment]
I am still having this issue on v 2.12
